ci-operator/step-registry/openshift/upgrade/qe/test/cert-rotation/openshift-upgrade-qe-test-cert-rotation-commands.sh

```diff
@@ -107,7 +107,7 @@ oc adm ocp-certificates regenerate-leaf -n openshift-config-managed secrets kube
 
 # distribute trust across all known clients
 # update our local CA bundle so that when new serving certs are used for kube-apiserver we will trust them
-oc config refresh-ca-bundle
+oc config refresh-ca-bundle || true
```

Comment:
Please add the following code to the line 129 as well, otherwise will hit the error `error: You must be logged in to the server (Unauthorized)`
```
oc --kubeconfig="${SHARED_DIR}/admin.kubeconfig" whoami || true
sleep 2
oc --kubeconfig="${SHARED_DIR}/admin.kubeconfig" whoami
```

Reply:
Thanks!